src/redux/reducers/store.js

```diff
@@ -4,21 +4,30 @@ import reduxThunk from 'redux-thunk';
 import logger from 'redux-logger';
 import rootReducer from './index';
 
+export const STORE_VERSION = '1';
+
 const saveState = (state) => {
   try {
     localStorage.setItem('state', JSON.stringify(state));
+    localStorage.setItem('storeVersion', STORE_VERSION);
     return undefined;
   } catch (e) {
     return undefined;
   }
 };
 
-const loadState = () => {
+export const loadState = () => {
   try {
     const state = localStorage.getItem('state');
     if (state === null) {
       return undefined;
     }
+
+    const storeVersion = localStorage.getItem('storeVersion');
+    if (storeVersion !== STORE_VERSION) {
+      return undefined;
```

Comment:
Now that the storeVersion is being stored simultaneously with the state, its value will always be equal to STORE_VERSION if the state was saved. I don't think this will ever be returned, because there's already an early return in place on line 23 incase there's no state persisted.

Reply:
The goal of this PR was to update the structure of the state when it changes in the code. So at the point when the code is loaded, `STORE_VERSION` in the code will be different from the `storeVersion` in local storage. This is the scenario where we want it to be cleared and updated.

Reply:
Okay